src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpContentDuplexStream.cs

```diff
@@ -12,6 +12,7 @@ internal abstract class HttpContentDuplexStream : HttpContentStream
     {
         public HttpContentDuplexStream(HttpConnection connection) : base(connection)
         {
+            if (NetEventSource.IsEnabled) NetEventSource.Info(this);
```

Comment:
Do we have similar tracing in the other abstract stream ctors?  This one is only used for 101 Switching Protocols right now.